capture.go

```diff
@@ -3,15 +3,16 @@ package main
 import (
```

Comment:
the only changes I made to this file are formatting and linting related, namely to shut my IDE up lmao.